deploy/image/ruby-bench-web/Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM samsaffron/discourse_base_22:1.0.0
+MAINTAINER Alan Guo Xiang Tan "https://twitter.com/tgx_world"
+
+RUN sudo apt-get update && sudo apt-get install -y qt5-default libqt5webkit5-dev
+
+RUN useradd rubybench -s /bin/bash -m -U &&\
+    mkdir -p /var/www && cd /var/www &&\
+      git clone https://github.com/ruby-bench/ruby-bench-web.git &&\
+      cd ruby-bench-web &&\
+        git config --global user.name "rubybench" &&\
+        git config --global user.email "https://twitter.com/tgx_world" &&\
```

Comment:
I ran into problems where it asked me for my `git config` when doing `git pull`.